src/dotnet/UI.Blazor.App/Components/ChatList/FoundResult/FoundPlaceListItem.razor

```diff
@@ -0,0 +1,100 @@
+@namespace ActualChat.UI.Blazor.App.Components
+@using ActualChat.Search
+@using SearchUI = ActualChat.UI.Blazor.App.Services.SearchUI
+@inherits ComputedRenderStateComponent<FoundPlaceListItem.Model>
+@{
+    var m = State.Value;
+    if (m == Model.None)
+        return;
+
+    var chatState = m.ChatState;
+    if (ReferenceEquals(m.Place, null) || m.Place.Id.IsNone)
+        return;
+}
+
+@if (m == Model.Loading) {
+    <ChatListItemSkeleton />
+} else {
+    <div class="found-result place">
+        <div class="c-content" data-href="@m.Link">
+            <div class="c-container">
+                <ChatIcon Chat="chatState.Chat"/>
```

Comment:
We should show place icon here, not chat icon. See NavbarPlaceButtons.
You can change now or later.
After this change ChatState will not be needed.

Reply:
I've changed it myself.